mscore/pianoview.cpp

```diff
@@ -691,7 +702,7 @@ void PianoView::mouseReleaseEvent(QMouseEvent* event)
                               if (!frac.isValid() || frac.isZero())
                                     frac.set(1, 4);
 
-                              if (cutChordRest(e, track, roundedTick, cr0, cr1)) {
+                              if (cutChordRest(e, track, Fraction(roundedTick, 1), cr0, cr1)) {
```

Comment:
Actually `Fraction::fromTicks()` should be used to convert from integer to fractional ticks, see e.g. line 677/678 above. Making a `Fraction(roundedTick, 1)` will yield a wrong result as fractional ticks are in "whole note" units which is not the case for integer ticks.

Reply:
Okay.  Do I need to squash this commit now or something?  I'm not sure how to incorporate changes from other users.

Reply:
There's only one commit, so nothing to squash

Reply:
I thini @dmitrio95 wants you to change `Fraction(roundedTick, 1)` to `fromTicks(roundedTick)`

Reply:
or just use `t` from line 677/678 above?

Reply:
Okay, just unsure about how git works here.  Does this comment indicate that @dmitrio95 has already added a patch, or do I need to add this on my end?

Reply:
Nevermind.  I saw the red line above and thought that this meant @dmitrio95 had submitted a patch.  I'll make the change.